src/manifest.rs

```diff
@@ -117,6 +124,28 @@ fn merge_dependencies(old_dep: &mut toml::value::Value, new: &Dependency) {
     }
 }
 
+fn print_upgrade_if_necessary(crate_name: &str, old_version: &toml::Value, new_version: &toml::Value) {
+    if old_version.as_str() == None || new_version.as_str() == None {
+        return
+    }
+    if old_version == new_version {
+        return
+    }
+
+    let bufwtr = BufferWriter::stdout(ColorChoice::Always);
+    let mut buffer = bufwtr.buffer();
+    buffer.set_color(ColorSpec::new().set_fg(Some(Color::Green)).set_bold(true)).unwrap();
+    write!(&mut buffer, "Upgrading ").unwrap();
+    buffer.set_color(&ColorSpec::new()).unwrap();
+    write!(&mut buffer,
+           "{} v{} -> v{}\n",
+           crate_name,
+           old_version.as_str().unwrap(),
+           new_version.as_str().unwrap()
+    ).unwrap();
+    bufwtr.print(&buffer).unwrap();
```

Comment:
Well, there are way too many `unwrap`s IMO but I don't know if returning a result s worth it